src/System.Net.Primitives/tests/FunctionalTests/IPAddressTest.cs

```diff
@@ -279,5 +279,26 @@ public static void Equals_Compare_Success()
             Assert.True(ip4.GetHashCode().Equals(ip5.GetHashCode()));
             Assert.False(ip4.GetHashCode().Equals(ip6.GetHashCode()));
         }
+
+#if NetStandard17
```

Comment:
cc: @weshaggard 
I believe we're aiming to put these tests into their own file that's conditionally compiled in.